rpmconf/Install/Util/globals.sh

```diff
@@ -38,7 +38,6 @@ zimbra-onlyoffice"
 SERVICES=""
 
 OPTIONAL_PACKAGES="zimbra-qatest \
-zimbra-imapd \
```

Comment:
at time of installation we also get question around if we want to install this component or not, have we removed that statement?

Reply:
once we remove zimbra-imapd from package list, we won't get option at the time of installation.